sos.spec

```diff
@@ -17,6 +17,7 @@ Requires: rpm-python
 Requires: tar
 Requires: bzip2
 Requires: xz
+Requires: python3-pexpect
```

Comment:
Ditto above. Maybe I just interpret Requirements wrongly.

On one side, this requirement is not necessary for `sos` in general, "just" for one its feature - so this requirement would be redundant. On the other side, functionality of whole `sos` package will be negatively affected, if this requirement is missing => it should be here.

(on the yet another side, `sos report` does not really require `xz` either way, but we have it among requirements as well - again, to ensure some particular functionality)

So if nobody else sees this as an issue, let ignore both comments.

Reply:
See above, same deal here - I think this should be up to the individual downstream packagers.